tests/integration_tests/base_tests.py

```diff
@@ -199,11 +199,8 @@ def get_or_create(self, cls, criteria, **kwargs):
     def login(self, username="admin", password="general"):
         return login(self.client, username, password)
 
-    def get_slice(self, slice_name: str, expunge_from_session: bool = True) -> Slice:
-        slc = db.session.query(Slice).filter_by(slice_name=slice_name).one()
-        if expunge_from_session:
-            db.session.expunge_all()
```

Comment:
It seems weird that we were expunging all objects from the session and not just the chart in question.